webapp/app/data_access/storage/cookie_storage.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional
+from flask import session
+
+from app.data_access.storage.form_storage import FormStorage
+
+
+class CookieStorage(FormStorage):
```

Comment:
Shouldn't we add the method docs in the super-class FormStorage? And remove them from the sub-class CookieStorage? The other sub-class SessionStorage doesn't have any docs

Reply:
Done